pkg/internal/client/lease.go

```diff
@@ -30,7 +30,7 @@ import (
 // DefaultLeaseDuration is the duration a lease will be acquired for if no
 // duration was specified in a LeaseManager's options.
 // Exported for testing purposes.
-const DefaultLeaseDuration = time.Minute
+const DefaultLeaseDuration = 1 * time.Minute
```

Comment:
The reason I made the change is because it's encouraged somewhere (having trouble finding it other than in https://blog.golang.org/go-concurrency-patterns-timing-out-and) to always multiply `time` constants by a number so that it reads correctly, even for `1 * time.Whatever`.